copy-reports/copy-reports.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+OLD_POD=$(kubectl get pod -n kubecost -l app=cost-analyzer -o jsonpath='{.items[0].metadata.name}')
+NEW_POD=$(kubectl get pod -n kubecost -l app=aggregator -o jsonpath='{.items[0].metadata.name}')
```

Comment:
I would add a check here that ensures their aggregator is deployed as a stateful set, as a guardrail